oauthlib/oauth2/rfc6749/request_validator.py

```diff
@@ -270,6 +277,20 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         """
         raise NotImplementedError('Subclasses must implement this method.')
 
+    def get_id_token(self, token, token_handler, request):
```

Comment:
Should this take a `nonce` explicitly? Or is it expected that implementors will get the nonce off the request object? If so, it might be good to document that.